b2/console_tool.py

```diff
@@ -344,6 +344,25 @@ def run(self, args):
         return 0
 
 
+class GetFileInfoAndDeleteFileVersion(Command):
```

Comment:
That is what I was trying to do in [this code snippet](https://github.com/Backblaze/B2_Command_Line_Tool/pull/227#discussion_r78472610)